client/webpack/fetch-translation-chunks-plugin.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * External dependencies.
+ */
+const webpack = require( 'webpack' ); //eslint-disable-line import/no-extraneous-dependencies
+
+const { Template } = webpack;
+
+/**
+ * Plugin name.
+ *
+ * @type {string}
+ */
+const PLUGIN_NAME = 'FetchTranslationChunks';
+
+class FetchTranslationChunksPlugin {
```

Comment:
Given the slightly changed function of the plugin, shall we rename it? e.g. `ChunkCallbackPlugin`?